src/views/Watch.vue

```diff
@@ -242,8 +243,8 @@ export default {
             // by default:
             //   mobile: not open
             //   desktop: open except in mugen (where TL doesnt work)
-            showTL: !this.$store.state.isMobile,
-            showTLFirstTime: !this.$store.state.isMobile,
+            showTL: vm.status === "live" && !this.$store.state.isMobile,
```

Comment:
Did you mean vm.video.status?

I would move both these variables into a computed value

Reply:
Yeah, my bad. I'll move them.